.github/workflows/publish.yml

```diff
@@ -7,10 +7,10 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ 'ubuntu-18.04', 'windows-2019', 'macos-latest' ]
+        os: ['ubuntu-22.04', 'windows-2019', 'macos-latest']
     steps:
       - name: Setup apt dependencies
-        if: matrix.os == 'ubuntu-18.04'
+        if: matrix.os == 'ubuntu-22.04'
```

Comment:
This is not going to work for all ExpressLRS users, because the most recent Ubuntu LTS release has a very recent libc version and people who use older linux distributions will not be able to use our application.

I think we could try updating to 20.04.